### PR TITLE
Added require.sh to check if composer is installed

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | 
| License       | MIT

As discussed in #2500, we need to check if composer is installed locally (with a `composer.phar` file). 
I added a `require.sh` file to check it.

### CODE REVIEW

File: scripts/require.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+# File used to check dependencies
+
+if [ ! -f composer.phar ]; then
+    echo "composer.phar not found, we'll see if composer is installed globally."
```
Comment (j0k3r) [inline scripts/require.sh]: if `composer.phar` exist the next command will still fail.

Comment (nicosomb) [inline scripts/require.sh]: Oops, you're right 😁

Comment (nicosomb) [inline scripts/require.sh]: and if `composer.phar` is found, we need to use it in the command, instead of `composer`.

**Comment by nicosomb** [comment]
I think it'sOK: variable seems to be global in bash.

**Pull request merged**